ast/misc.go

```diff
@@ -398,6 +422,13 @@ func (n *BeginStmt) Accept(v Visitor) (Node, bool) {
 		return v.Leave(newNode)
 	}
 	n = newNode.(*BeginStmt)
+	if n.Bound != nil && n.Bound.Timestamp != nil {
+		newTimestamp, ok := n.Bound.Timestamp.Accept(v)
+		if !ok {
+			return n, false
+		}
+		n.Bound.Timestamp = newTimestamp.(ExprNode)
```

Comment:
I guess this should work:

`n.Bound.Timestamp = newTimestamp`

Reply:
nope, it complains😂
> Node does not implement ExprNode (missing Format method)